docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -63,14 +63,37 @@ Set the Elasticsearch settings before deploying the manifest:
   value: "https://somesuperhostiduuid.europe-west1.gcp.cloud.es.io:443"
 ------------------------------------------------
 
+// Begin collapsed section
+[%collapsible]
+.Configuration details
```

Comment:
The configuration names in the example don't match the reference info: `ES_USERNAME` vs `ELASTICSEARCH_USERNAME`

Also might be worth stating what takes precedence: the settings in the manifest or the env vars.

Reply:
Oh wow, I totally missed that. I'll need someone from engineering to follow-up with on this.